XWSBack/BaseApi.Services/Exceptions/BadRequestException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BaseApi.Services.Exceptions
+{
+    public class BadRequestException : Exception
```

Comment:
Servisni sloj ne bi trebalo da zna ko ga koristi i ko ga poziva. U prevodu lose je ime, ja bih stavio "ValidationException" jer to je nesto sto se radi svuda